### DISCUSSION TITLE
如何在集群中使用xmake？

### DISCUSSION THREAD

**Discussion by MIngPAPA** [xmake-io/xmake#3104] [Q&A] [answered]
根据Xmake Doc目前的xmake的实践不推荐root安装。

我有一个cluster，多用户同时使用。
如何像cmake、make一样安装一个“集中式”的xmake程序让所有人都可以直接使用呢？

**Comment by waruqi** [comment, marked as answer]
> 根据Xmake Doc目前的xmake的实践不推荐root安装。

不是安装，主要是不推荐 root 用户运行使用，你可以 root 装完，切到非 root 用户下去用。。
